index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portofolio page</title>
+    <link rel="stylesheet" href="style.css"> 
+</head>
+<body>
+    <header class="container">
+        <nav class="nav">
+        <a class="my-logo" href="index.html">
+        <img src="assets/images/My%20Logo.svg" alt="Portofolio logo"> 
+        </a>
+        <a class="hamburger" href="#">
+        <img src="assets/images/burger.svg" alt="Hambugrer_menu">   
+        </a>
+    </nav>
```

Comment:
- [ ] You're including both the logo and the hamburger icon in the `<nav>` tag, but it should only include elements that include navigation elements. As only the hamburger element will contain the menu items later on in the project, please change your codebase to include only the hamburger icon in the `<nav>` tag. The logo is fine as it is, good job making it a link to the homepage 👍